consensus/pbft/batch.go

```diff
@@ -261,31 +228,16 @@ func (op *obcBatch) leaderProcReq(req *Request) events.Event {
 
 func (op *obcBatch) sendBatch() events.Event {
 	op.stopBatchTimer()
-
 	if len(op.batchStore) == 0 {
 		logger.Error("Told to send an empty batch store for ordering, ignoring")
 		return nil
 	}
 
-	earliestRequest := op.batchStore[0]
-
-	reqBlock := &RequestBlock{op.batchStore}
+	reqBatch := &RequestBatch{Batch: op.batchStore}
 	op.batchStore = nil
-
-	reqsPacked, err := proto.Marshal(reqBlock)
-	if err != nil {
-		logger.Error("Unable to pack block for new batch request")
-		return nil
-	}
-
-	// process internally
-	logger.Infof("Creating batch with %d requests", len(reqBlock.Requests))
+	logger.Infof("Creating batch with %d requests", len(reqBatch.Batch))
 	return pbftMessageEvent{
-		msg: &Message{&Message_Request{&Request{
-			Payload:   reqsPacked,
-			Timestamp: earliestRequest.Timestamp,
-			ReplicaId: op.pbft.id},
-		}},
+		msg:    &Message{Payload: &Message_RequestBatch{RequestBatch: reqBatch}},
```

Comment:
Is this necessary to wrap in a message? Could this nor be returned simply as a `RequestBatch`?